plan/service.py

```diff
@@ -124,15 +124,6 @@ def start_trial(self) -> None:
         self.current_org.plan_auto_activate = True
         self.current_org.save()
 
-        self.notifier_service.trial_started(
```

Comment:
Maybe you can also delete `self.notifier_service` in line 26. It's not being used anymore, apparently